model/src/main/kotlin/PackageCuration.kt

```diff
@@ -27,6 +27,11 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
 
 import org.semver4j.Semver
 
+/**
+ * A list of Strings, that are used to identify a version string as a version range in the [PackageCuration]'s version.
+ */
+private val versionRangeIndicators = listOf(",", "~", "*", "+", ">", "<", "=", " - ", "^", ".x", ".X")
```

Comment:
Since https://github.com/oss-review-toolkit/ort/pull/8142cdc06a871bc1e66d4b2df82ffcf0fbd20ee5 we implicitly support every version range specified as a `Processor` in semver4j/semver4j [1].
If we want to disable this, and only support Ivy ranges [2] has to be implemented.

[1]: https://github.com/semver4j/semver4j/blob/main/src/main/java/org/semver4j/internal/range/processor/Processor.java
[2]: https://github.com/semver4j/semver4j/issues/133